src/main/resources/static/assets/css/main.css

```diff
@@ -91,4 +91,8 @@ td.table-success {
   color: #bd3737;
 }
 
+.fas.fa-save {
+	color: #007bff;
```

Comment:
It might be more consistent with Bootstrap to make this icon the same blue as `btn-primary`, rather than green.

Reply:
I actually did. #007bff is the same as text-primary in bootstrap.